package.json

```
@@ -34,7 +34,7 @@
     "typescript": "^4.4.2"
   },
   "dependencies": {
-    "@pooltogether/v4-mainnet": "1.1.0",
+    "@pooltogether/v4-mainnet": "1.4.0-beta.2",
     "@pooltogether/v4-testnet": "1.1.0-beta.2"
```

We could update testnet to `1.4.0`.
```suggestion
    "@pooltogether/v4-testnet": "1.4.0"
```